src/biogeophys/TemperatureType.F90

```diff
@@ -141,7 +141,7 @@ module TemperatureType
   !------------------------------------------------------------------------
   subroutine Init(this, bounds, &
```

Comment:
As we want to read the namelist here, you'll need to add NLFilename to the arguments, here and in the call.